packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -148,8 +152,23 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
     if (this.props.parentRef) {
       this._fitParentContainer();
     }
+
+    const margin = { ...this.margins };
+    if (this.props.showYAxisLables) {
+      const maxYAxisLabelLength = calculateLongestYAxisLabel(points, this.props.theme!);
```

Comment:
This is not a property of Cartesian Chart. This should be limited to Horizontal Bar Chart with axis

Reply:
Lets put this in horizontal bar chart with axis for now.

Reply:
This is done.